frontend/pages/admin/TeamManagementPage/components/DeleteTeamModal/DeleteTeamModal.tsx

```diff
@@ -31,8 +31,8 @@ const DeleteTeamModal = ({
           <span className={`${baseClass}__name`}>{name}</span> from Fleet.
         </p>
         <p>
-          Members of this team who are not members of other teams will lose
-          access to Fleet.
+          Users on team who are not assigned to other teams will lose access to
```

Comment:
@RachelElysia I realize this was already merged but had this open and realized this sentence is missing a word (should be "Users on **this** team")